src/App.jsx

```diff
@@ -5,24 +5,33 @@ import ExponentThree from "./components/ExponentThree";
 import ExponentFour from "./components/ExponentFour";
 import ExponentFive from "./components/ExponentFive";
 import ExponentSix from "./components/ExponentSix";
-
+import { useState } from "react";
+import ExponentN from "./components/ExponentN";
 
 function App () {
+  
+  const [count, setCount] = useState(1);
```

Comment:
"Lifting the state up". Ahora el estado puede ser compartido con los otros elementos que lo necesiten. En este caso todos los Exponent components.